### DISCUSSION TITLE
German Student ID + Zeugnis not accepted

### DISCUSSION THREAD

**Discussion by Simoso68** [community/community#129194] [GitHub Education]
Hello! :wave:

I've got a problem with my "Github Student Benefit" request being rejected.
The proof includes my german student ID card, to be more exact, an ID card from the state of Thuringia
and it includes my Zeugnis (a paper to conclude both semesters with my grades).

The ID card includes my full name, date of birth, place of residence, the school I go to and a picture of my face (around 2 years old)

The Zeugnis includes my current grade, years of both semesters, full name, date of birth and the school I go to

I selected "Other" as proof type.

I also changed my display name to my real full name, as it was requested.

In my eyes, this should be enough and I can't imagine, how this information can't be used to approve me as a legitimate student.

If someone has any idea, on how one could help me out, please share it.
Kind regards, Simon

**Comment by sachetsubedi** [comment]
You need to have a school  mail account to be recognized as a student even you have every piece of paper. I think it is done because fake id proofs can be submitted. 
If you have the mail account configured and selected, maybe other details like billing details and 2F is not configured properly which is required

**Reply by Simoso68** [reply]
My school does not provide E-Mail accounts.
Only teachers get a mail for their profession, but those are handed out by the state of Thuringia.

Regarding my billing information, there were a few optional fields, that I haven't filled out at the time of my request (State/Province and ZIP Code).
I've added these now and I'll try to reapply in a moment.

**Reply by sachetsubedi** [reply]
You are required to have the student mail account, and school must provide it to its students. I see no other options than buying the pro pack

**Reply by Simoso68** [reply]
Yeah, just tried to reapply and got rejected instantly.
The only thing close to a mail provided by my school, is the mail, that comes with a "Thüringer Schulcloud" (engl.: Thuringian school cloud) account.
The account was created by my school, but it the Mail-Account isn't directly offered by my school, so it might or might not be valid.

**Reply by Simoso68** [reply]
Nevermind, I can't even log into that E-Mail, seems like a mail, that can't be used.
Sadly, my school never provided an E-Mail, that I could use.
And GitHub does not allow E-Mail support for help regarding applications for the Student Benefit Program.
I also don't think, that I can get in touch with an employee at GitHub that easily regarding this topic.
